src/aiperf/common/config/audio_config.py

```
@@ -139,4 +146,18 @@ class AudioConfig(BaseConfig):
         ),
-    ] = AudioDefaults.NUM_CHANNELS
+    ]
+
+    @model_validator(mode="after")
+    def _validate_audio_options(self) -> Self:
+        """Validate the audio options."""
+        audio_options_set = {*self.model_fields_set, *self.length.model_fields_set}
+        if not self.audio_enabled() and audio_options_set:
+            raise ValueError(
+                "Audio generation is disabled but audio options were provided. Please set `--audio-batch-size` and `--audio-length-mean` to enable audio generation."
+            )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)